analyses/create-subset-files/create_subset_files.sh

```diff
@@ -68,9 +68,6 @@ cp $FULL_DIRECTORY/independent-specimens*.tsv $SUBSET_DIRECTORY
 # all bed files
 cp $FULL_DIRECTORY/*.bed $SUBSET_DIRECTORY
 
-# the release notes
-cp $FULL_DIRECTORY/release-notes.md $SUBSET_DIRECTORY
```

Comment:
Ah, this is why I was missing files. Because the script failed at this step before and the other files after it did not get copied. I ran everything again just in case.